gator/fragments.py

```diff
@@ -128,13 +138,13 @@ def specified_fragment_greater_than_count(
     contents=NOTHING,
     exact=False,
 ):
-    """Determines if the fragment count is greater than expected"""
-    # count the fragments in either a file in a directory or String contents
-    file_fragment_count = count_fragments(
+    """Determines if the entity count is greater than expected"""
+    # count the fragments/regex in either a file in a directory or String contents
+    file_entity_count = count_fragments(
```

Comment:
`count_fragments` should also be fixed to express its use here -- `count_entities` may be better.

Reply:
@Michionlion Thanks for the suggestion, I have made this change across the necessary files.